src/msg/async/EventKqueue.cc

```diff
@@ -199,13 +199,17 @@ int KqueueDriver::resize_events(int newsize)
   ldout(cct,30) << __func__ << " kqfd = " << kqfd << "newsize = " << newsize 
                 << dendl;
   if(newsize > sav_max) {
-    sav_events = (struct SaveEvent*)realloc( sav_events, 
-                    sizeof(struct SaveEvent)*newsize);
-    if (!sav_events) {
+    void *_realloc = NULL;
+    _realloc = realloc(sav_events,
+                          sizeof(struct SaveEvent)*newsize);
+    if (_realloc == NULL) {
```

Comment:
@tchaikov I will make the change.
It seems the caller is at
`  // keep what we set
  if (fd >= sav_max)
    resize_events(sav_max+5000);`
in
`int KqueueDriver::add_event(int fd, int cur_mask, int `add_mask)`